src/utils/CaptainConstants.ts

```diff
@@ -87,7 +90,7 @@ const data = {
 
     isDebug: EnvVars.CAPTAIN_IS_DEBUG,
 
-    captainServiceExposedPort: 3000,
+    captainServiceExposedPort: configs.adminPortNumber3000,
```

Comment:
this variable is just now a mirror for `configs.adminPortNumber3000` - should we remove it completely and replace the callsites with `CaptainConstants.configs.adminPortNumber3000` ?

Reply:
Removed and used in all callsites `CaptainConstants.configs.adminPortNumber3000`.